src/amoc.erl

```diff
@@ -32,6 +30,7 @@ do(Scenario, Count, Settings) ->
         Error -> Error
     end.
 
+%% @doc dynamically add more users to a currently running scenario
```

Comment:
Some of the docs are starting with a capital letter and some don't. Please make it consistent for all modules. I think capital letter is nicer.